app/Models/Entry.php

```diff
@@ -663,6 +663,9 @@ public function toGReader(string $mode = ''): array {
 		$feed = $this->feed();
 		$category = $feed == null ? null : $feed->category();
 
+		// Some clients (tested with News+) would fail if sending too long item content
+		$maxCompatContentLength = 500000;
```

Comment:
I would rather use a class constant. It will work the same but I think it's cleaner since it cannot be modified by the code.

Reply:
Done https://github.com/FreshRSS/FreshRSS/pull/4583/commits/8b99285c067cd5a64ea3012d0062a05f2ace158f